_includes/guides/create-dockerfile.md

```diff
@@ -0,0 +1,39 @@
+A Dockerfile is a text document that contains the instructions to assemble a
+Docker image. When we tell Docker to build our image by executing the `docker build`
+command, Docker reads these instructions, executes them, and creates a Docker
+image as a result.
```

Comment:
We may want to touch-up this section a bit; some versions of this section described it as "executed in sequence" or "one-by-one". With BuildKit, steps in a Dockerfile can be executed in parallel in some cases, so it's not all "sequential".

Explaining "building a graph" etc. may be out of scope for this though, so I kept it as-is, but perhaps we can find a way to include that information (or link to somewhere that describes it)